Dockerfile.glibc

```diff
@@ -37,26 +37,32 @@ ENV  USER=iroha
 ENV  UID=1001
 ENV  GID=1001
 
-RUN  set -ex && \
-     apt-get update -y && \
-     apt-get install -y curl ca-certificates jq && \
-     addgroup --gid $GID $USER && \
-     adduser \
-     --disabled-password \
-     --gecos "" \
-     --home /app \
-     --ingroup "$USER" \
-     --no-create-home \
-     --uid "$UID" \
-     "$USER" && \
-     mkdir -p $CONFIG_DIR && \
-     mkdir -p $STORAGE && \
-     mkdir -p $WASM_DIRECTORY && \
-     chown $USER:$USER $STORAGE && \
-     chown $USER:$USER $WASM_DIRECTORY
+RUN <<EOT
+  set -ex
+  apt-get update -y && \
+    apt-get install -y curl ca-certificates jq
+  addgroup --gid $GID $USER &&
```

Comment:
```suggestion
  apt-get update -y
  apt-get install -y curl ca-certificates jq
  addgroup --gid $GID $USER
```

Reply:
This whole block is already treated like one operation because of `set -e` short-circuiting on failure